### ISSUE TITLE
Shared Service_Address masks get clobbered

### ISSUE THREAD

**Issue by wncocz** [F5Networks/f5-appsvcs-extension#339]
### Environment
 * Application Services Version: 3.22.1
 * BIG-IP Version: 13.1.3

### Summary

Shared Service_Address objects easily get their netmasks clobbered by AS3 when deploying virtuals that reference them.

### Steps To Reproduce

We created a Shared declaration that creates:
```
"vs_0.0.0.0_0": {
          "class": "Service_Address",
          "virtualAddress": "0.0.0.0/0",
          "arpEnabled": false,
          "icmpEcho": "disable",
}
```

We deployed that, and it looks fine:

```
# list ltm virtual-address /Common/Shared/va_0.0.0.0_0
ltm virtual-address va_0.0.0.0_0 {
    address any
    arp disabled
    icmp-echo disabled
    inherited-traffic-group true
    mask any
    traffic-group /Common/traffic-group-1
}
```

Then we deployed the passthru Tenant: with a virtual like so:

```
                "passthru_from_10.11.12.0_24.oc.w11_0": {
                    "allowVlans": [ { "bigip": "/Common/inside" } ],
                    "class": "Service_Forwarding",
                    "forwardingType": "ip",
                    "profileL4": { "bigip": "/Common/fastL4" },
                    "snat": "none",
                    "translateServerPort": false,
                    "virtualAddresses": [
                        [
                            { "bigip": "/Common/Shared/va_0.0.0.0_0" },
                            "10.11.12.0/24"
                        ]
                    ],
                    "virtualPort": 0
                },
```

### Expected Behavior

AS3 should fetch the mask from the referenced virtual-address and use that when deploying the virtual.

### Actual Behavior

The cli script generated has ``mask 255.255.255.255`` which causes our /Common/Shared/va_0.0.0.0 to get assigned a mask of 255.255.255.255.  This breaks other virtuals that are using the same virtual-address!

After deployment, we must manually ``modify ltm virtual /T/A/virtualname mask any``

This is not unique to the address 0.0.0.0/0.  We tried with 128.0.0.0/1, 10.0.0.0/8, etc.  Mask is always reset to 255.255.255.255 when deploying a virtual that uses the shared Service_Address object.

**Comment by dstokesf5** [comment]
I have confirmed your bug and added it to our internal product backlog as AUTOTOOL-1885.

**Comment by dstokesf5** [comment]
It appears that this works correctly for use-pointers. In order to make use of a use-pointer, the virtual address and virtual server need to be part of the same declaration. We are continuing to look into having this work with bigip pointers as well.

**Comment by dstokesf5** [comment]
This was resolved in the 3.24.0 release.
